### PR TITLE
feat: improve order status method

### PR DESCRIPTION
**Pull request by mvarlic**
* improve order status method
* Status OK
image
* Status Error
image

### CODE REVIEW

File: plugin/views/get-status.php [hunk 1]
```diff
@@ -109,3 +109,7 @@
 
  </table>
 
+ <div class="error-transaction-status-response" cellspacing="0" cellpadding="0" style="display: none">
+    <div style="font-weight: bold;">Error consultando estado de la transacción</div>
+    <div class="error-status-raw"></div>
+</div>
```
Comment (victormendoza96) [inline plugin/views/get-status.php:115]: missing empty line

File: plugin/views/get-status.php [hunk 2]
```diff
@@ -109,3 +109,7 @@
 
  </table>
 
+ <div class="error-transaction-status-response" cellspacing="0" cellpadding="0" style="display: none">
+    <div style="font-weight: bold;">Error consultando estado de la transacción</div>
+    <div class="error-status-raw"></div>
```
Comment (victormendoza96) [inline plugin/views/get-status.php:114]: why style in line?

**Review by victormendoza96 — CHANGES_REQUESTED** [review]
[feat: rermove console log](https://github.com/TransbankDevelopers/transbank-plugin-woocommerce-webpay-rest/pull/154/commits/65432ff7b5fd41b6e7384ffe25808dbd5142a3b0)
this commit is a "refactor" not a "feat"

File: plugin/views/get-status.php (hunk 2 above)
Comment (mvarlic) [reply plugin/views/get-status.php]: moved to css file

**Review by victormendoza96 — APPROVED** [review]

**Pull request merged**